src/app/editor-common/app-editor-base/app-editor-base.ts

```diff
@@ -343,6 +351,10 @@ export abstract class AppEditorBase extends DirectiveBase implements StatusContr
 
 
                 this.statusBar.stopProcess(proc, `Saved: ${appName}`);
+
+                if (this.tabData.dataSource === "local") {
+                    this.updateService.updateApps({id: this.tabData.id, app: Yaml.safeLoad(update, {json: true} as LoadOptions)});
```

Comment:
@ivanbatic's suggestion to wrap the parse/Yaml.safeLoad into a try/catch block applies here, too.